.circleci/config.yml

```
@@ -32,13 +32,13 @@ commands:
       - node/install-packages:
           pkg-manager: yarn
   db-setup:
-    description: Set up database
+    description: Create database
     steps:
       # Using structure.sql? Install postgresql-client for loading structure.
       #- run: sudo apt-get install postgresql-client
       - run:
-          name: Set up database
-          command: bundle exec rake db:setup
+          name: Create database
+          command: bundle exec rake db:create
```

If we'd rather not rely on RSpec implicitly loading the schema, we could do it explicitly instead:

```suggestion
          command: bundle exec rake db:schema:load
```

Which is what the CircleCI documentation recommends: https://circleci.com/docs/2.0/language-ruby/#sample-configuration

However if we load the schema, that prevents us from being able to test migrations (running migrations against a fully-loaded schema is a no-op), but maybe testing migrations is not needed/desired ([see my other comment](https://github.com/carbonfive/raygun-rails/pull/666/files#r809611633)).